pkg/virt-operator/resource/generate/components/scc.go

```diff
@@ -74,6 +74,9 @@ func NewKubeVirtControllerSCC(namespace string) *secv1.SecurityContextConstraint
 	scc.SELinuxContext = secv1.SELinuxContextStrategyOptions{
 		Type: secv1.SELinuxStrategyRunAsAny,
 	}
+	scc.SeccompProfiles = []string{
+		"runtime/default",
```

Comment:
Shouldn't it be `RuntimeDefault`?

Reply:
nope, check out the snippet from a running cluster in the desc